### ISSUE TITLE
Converting a string containing \"

### ISSUE THREAD

**Issue by lorcQc** [paul1956/CSharpToVB#63]
When I paste this code in the converter :
`test1 = "Simple\"Test";`

If the  "colorize source" option is active, the code is transformed into :
`test1 = "Simple\"Test"\"Test";`

Regardless of the option "colorize source" is active or not, the generated VB code is incorrect.
`test1 = "Simple""Test"""Test"`

**Comment by paul1956** [comment]
@lorcQc I saw that when the colorizer is on. There is """ and some extra text. This is coming from the colorizer and I have not yet figured out why. The version I just posted I turn off the colorizer for many scenarios which helps. But I need to figure would what is happening, the colorize code is something I have will the original repo and I haven't changed it but maybe on the the Roslyn libraries has changed. I will look at this today.

With the colorizer off it looks to me to be generating the correct result
```
test1 = "Simple""Test"
```
